app/views/userInterface/index.ejs

```diff
@@ -13,8 +13,33 @@
         <script src="javascripts/materializejs/materialize.min.js"></script>
         <link href="stylesheet/datatable/datatable.min.css" rel="stylesheet">
         <link href="stylesheet/dashboard.css" rel="stylesheet" type="text/css">
+        <link href="stylesheet/deleteModal.css" rel="stylesheet">
         <link href="stylesheet/clientsModal.css" rel="stylesheet">
         <link href="stylesheet/notifModal.css" rel="stylesheet">
+
+        <style>
```

Comment:
Maybe put the style into its own css file?

Reply:
just moved it, let me know if that works